core/controllers/suggestion.py

```diff
@@ -302,6 +302,61 @@ class SuggestionListHandler(base.BaseHandler):
     """Handles list operations on suggestions."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'suggestion_type': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'choices': feconf.SUGGESTION_TYPE_CHOICES,
+                'default_value': None
+            },
+            'target_type': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'choices': feconf.SUGGESTION_TARGET_TYPE_CHOICES,
+                'default_value': None
+            },
+            'target_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'status': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'author_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'final_reviewer_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'score_category': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
```

Comment:
What are the possible score categories?

Reply:
After doing some research it seems like score categories are of the format 
<SCORE_TYPE_CHOICE>.<Subcategory_of_suggestion>.
I was wondering 
1. If there a way to find possible subcategories of a suggestion?
2. Would I have to write a new validator for this format?